src/invalidations.jl

```diff
@@ -255,6 +255,41 @@ else
     new_backedge_table() = Dict{Tuple{Int32,UInt64},Tuple{Any,Vector{Any}}}()
 end
 
+"""
+    report_invalidations(;
+        job_name::String = "",
+        invalidations,
+        n_rows::Int = 10,
+        process_filename::Function = x -> x,
+    )
+
+Print a tabular summary of invalidations given:
+
+ - `job_name::String` the name of the job
+
+An `@info` message also prints some information, for example
+if the table has been truncated (upon request).
```

Comment:
This might be interpreted as saying "you have to request the `@info` message" whereas I think you mean "it was truncated by request," which in turn really means "it was truncated by specifying `nrows`."

But even this is confusing since `nrows` has a default value of 10, so you don't actually have to request it.